.github/workflows/build-macos-installers.yml

```
@@ -450,6 +450,11 @@ jobs:
         run: |
           find "/Volumes/Chia "*
 
+      - name: Test for homebrew rpath
+        if: matrix.arch.name == 'ARM64'
```

This is what the `.matrix` elements are for.  This way when we bicker about what the name output should be we don't have to update all the references in the workflow.

```suggestion
        if: matrix.arch.matrix == 'arm'
```

Or, does running this in the Intel case cause trouble?

```suggestion
```

---

I'll propose my changes in another PR to keep this moving.